frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditDate/EditDate.tsx

```diff
@@ -28,9 +40,33 @@ import { FormFieldDrawerActions } from '../common/FormFieldDrawerActions'
 import { EditFieldProps } from '../common/types'
 import { useEditFieldForm } from '../common/useEditFieldForm'
 
+export interface InvalidDaysCheckboxOptions {
+  label: string
+  value: string
+}
+
+const INVALID_DAYS_CHECKBOX_VALUES: InvalidDaysCheckboxOptions[] = [
```

Comment:
it they are the same value, do you think we can get away with just the enums in an array? 

```tsx
const INVALID_DAYS_CHECKBOX_VALUES = [InvalidDaysOptions.Monday, ...]
```

Reply:
Yeap I guess we can just map over the values in the enum when creating the checkboxes

Reply:
probably need a constant to preserve order (current enum ordering is in order (for now) but it's not actually guaranteed)